src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
I would do `itemlist = list(heights.items())`. In terms of performance, it's a small bullet to bite. This would mean that only python 2 users need to install future. What do you think? Problems in the autoconf? conda packaging?